### PR TITLE
FrmMain Changes on Model Menu

### PR DESCRIPTION
**Pull request by Wycklife**
@africanmathsinitiative/developers this is ready for review.

### CODE REVIEW

File: instat/frmMain.vb [hunk 1]
```diff
@@ -372,7 +372,7 @@ Public Class frmMain
         dlgRestrict.ShowDialog()
     End Sub
 
-    Private Sub mnuModelThreeVariablesFitModel_Click(sender As Object, e As EventArgs) Handles mnuModelThreeVariablesFitModel.Click
+    Private Sub mnuModelThreeVariablesFitModel_Click(sender As Object, e As EventArgs)
```
Comment (maxwellfundi) [inline instat/frmMain.vb]: These empty subs needs to be removed @Wycklife

File: instat/frmMain.vb [hunk 2]
```diff
@@ -715,7 +715,7 @@ Public Class frmMain
         dlgRandomSample.ShowDialog()
     End Sub
 
-    Private Sub mnuModelTwoVariablesFitModel_Click(sender As Object, e As EventArgs) Handles mnuModelTwoVariablesFitModel.Click
+    Private Sub mnuModelTwoVariablesFitModel_Click(sender As Object, e As EventArgs)
```
Comment (maxwellfundi) [inline instat/frmMain.vb]: This one too

File: instat/frmMain.Designer.vb [hunk 3]
```diff
@@ -92,31 +92,27 @@ Partial Class frmMain
...
+        Me.mnuModelFitModelFourVariables = New System.Windows.Forms.ToolStripMenuItem()
+        Me.ToolStripSeparator24 = New System.Windows.Forms.ToolStripSeparator()
+        Me.mnuModelFitModelGeneral = New System.Windows.Forms.ToolStripMenuItem()
+        Me.ToolStripSeparator57 = New System.Windows.Forms.ToolStripSeparator()
+        Me.mnuModelFitModelModelKeyboard = New System.Windows.Forms.ToolStripMenuItem()
         Me.mnuModelUseModel = New System.Windows.Forms.ToolStripMenuItem()
+        Me.mnuModelUseModelOneVariable = New System.Windows.Forms.ToolStripMenuItem()
+        Me.mnuModelUseModelTwoVariables = New System.Windows.Forms.ToolStripMenuItem()
+        Me.ToolStripSeparator59 = New System.Windows.Forms.ToolStripSeparator()
+        Me.mnuModelUseModelGlance = New System.Windows.Forms.ToolStripMenuItem()
```
Comment (maxwellfundi) [inline instat/frmMain.Designer.vb]: @Wycklife  i think the new Glance, Tidy and Augment Menus needs to be diasbaled untill they have dialogs merged and added to form main

File: instat/frmMain.vb [hunk 4]
```diff
@@ -2118,4 +2114,44 @@ End Sub
...
+
+    Private Sub OneVariableToolStripMenuItem1_Click(sender As Object, e As EventArgs) Handles mnuModelUseModelOneVariable.Click
+        dlgOneVarUseModel.ShowDialog()
+    End Sub
+
+    Private Sub TwoVariablesToolStripMenuItem1_Click(sender As Object, e As EventArgs) Handles mnuModelUseModelTwoVariables.Click
+        dlgTwoVariableUseModel.ShowDialog()
+    End Sub
+
+    Private Sub UseModelKeyboardToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles mnuModelUseModelUseModelKeyboard.Click
```
Comment (maxwellfundi) [inline instat/frmMain.vb]: The sub names for these needt to be more informative @Wycklife , You can change these and it all looks good.

**Comment by Wycklife** [comment]
Okay, let me work on those changes @maxwellfundi

**Comment by Wycklife** [comment]
I have made suggested changes @maxwellfundi

**Comment by maxwellfundi** [comment]
Looks good!

**Pull request merged**